package.json

```diff
@@ -1288,27 +1288,38 @@
     "contributors": "./scripts/generate_contributors_list.sh"
   },
   "devDependencies": {
+    "@types/chai": "^4.3.19",
+    "@types/chai-as-promised": "^7.1.8",
+    "@types/chai-subset": "^1.3.5",
     "@types/glob": "^7.1.6",
     "@types/lcov-parse": "^1.0.2",
     "@types/mocha": "^10.0.7",
+    "@types/mock-fs": "^4.13.4",
     "@types/node": "^18.19.39",
     "@types/plist": "^3.0.5",
+    "@types/sinon": "^17.0.3",
+    "@types/sinon-chai": "^3.2.12",
     "@types/vscode": "^1.88.0",
     "@types/xml2js": "^0.4.14",
     "@typescript-eslint/eslint-plugin": "^7.16.0",
     "@typescript-eslint/parser": "^7.16.0",
     "@vscode/test-cli": "^0.0.10",
     "@vscode/test-electron": "^2.4.1",
     "@vscode/vsce": "^2.30.0",
+    "chai": "^4.5.0",
+    "chai-as-promised": "^7.1.2",
+    "chai-subset": "^1.6.0",
     "del-cli": "^5.1.0",
     "esbuild": "^0.23.0",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
     "mocha": "^10.6.0",
+    "mock-fs": "^5.2.0",
```

Comment:
Looks like there is only one example of this. So not convinced that adding is necessary, at the very least we need to add this to the writing effective unit tests docs and make clear why using the other utilities is not sufficient

Reply:
Mocking the global module fs also works, for example:
https://github.com/michael-weng/vscode-swift/blob/9204486e4704a4544cd7a14af9817a79e971f116/test/unit-tests/debugger/lldb.test.ts

Reply:
Yeah, it's not a great example since that one can be easily replaced with `mockGlobalModule()`. I've removed `mock-fs` for now until we can find a proper use case for it.